styles/components/_selectize.scss

```diff
@@ -0,0 +1,48 @@
+#ov1 {
+  .selectize-control {
+    .selectize-input {
+      font-size: inherit;
+      padding-top: 7px;
+      min-height: $formElementHeight;
+
+      input {
+        font-size: inherit;
+      }
+
+      &:not(.has-items) input {
+        top: 3px;
```

Comment:
This should have position: relative. Otherwise the top property doesn't get applied until after the input is in focus, which applies an inline position:relative, and that causes the placeholder to jump down.

Reply:
fixed